openedx_ledger/admin.py

```diff
@@ -303,17 +330,24 @@ def get_readonly_fields(self, request, obj=None):
         """
         if obj and not obj._state.adding:  # pylint: disable=protected-access
             return ['ledger'] + self._readonly_fields
-        return self._readonly_fields
+        return []
```

Comment:
[nit/curious] Wondering if an empty array return is intentional here?

Reply:
Good catch, yes I think this is leftover from testing and reverting it does not appear to impact django admin usability.